### ISSUE TITLE
Consider "families" of browsers

### ISSUE THREAD

**Issue by miketaylr** [whatwg/compat#177]
Right now we talk about Chrome, Firefox, Safari... but there are obviously other browsers like Edge, Opera, Waterfox, Epiphany, etc. We should look to see if they follow a stable pattern and document it if so.

**Comment by karlcow** [comment]
@gsnedders yes we were saying during the meeting than giving examples only for Safari, Chrome and Firefox and not other browsers seem a bit strange. So we were wondering if there was a larger pattern where we could add a keyword for specializing the UA string.

For example for Chromium based browser, there is an additional token at the end of the string.

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4851.0 Safari/537.36 Edg/100.0.1158.0
```

For Gecko based browsers, there is also a similar pattern 

```
Mozilla/5.0 (X11; FreeBSD amd64; rv:56.0) Gecko/20100101 Firefox/56.0 Waterfox/56.2.14
```

**Comment by gsnedders** [comment]
For WebKit products:

 * https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/ios/UserAgentIOS.mm
 * https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/mac/UserAgentMac.mm
 * https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/win/UserAgentWin.cpp
 * https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/glib/UserAgentGLib.cpp
 * https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/playstation/UserAgentPlayStation.cpp

This shows quite a lot of variety here, even before you get into application name suffixes.

**Comment by karlcow** [comment]
I think @gsnedders recently looked at the UA strings for different apps using WKWebView.
Could you share here what you got in patterns?
